samples/src/test/java/ua/net/tokar/json/filters/sample/ApplicationApiCompositionTest.java

```diff
@@ -85,4 +85,27 @@ public void deepInclusionTest() throws Exception {
         assertThat( body, hasNoJsonPath( "$[0].users[0].friends.href" ) );
         assertThat( body, hasNoJsonPath( "$[0].users.href" ) );
     }
+
+    @Test
+    public void includeWithRequestParamsTest() throws Exception {
+        String usersParams = "{offset:1,limit:2}";
```

Comment:
please inline these variables into queryString for better readability of the test

Reply:
cant, because curly braces in restTemplate means url vars

Reply:
can they be escaped? that still will be more convenient than variables